crates/oxc_linter/src/snapshots/prefer_es_6_class.snap

```diff
@@ -0,0 +1,36 @@
+---
+source: crates/oxc_linter/src/tester.rs
+---
+  ⚠ eslint-plugin-react(prefer-es6-class): Component should use createClass instead of es6 class
```

Comment:
I think you have these diagnostics backwards

Reply:
Yup, fixed and also rewrote `with_labels` to `with_label`. Thanks for spotting.